test/libsolidity/syntaxRecoveryTests/e1.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity >=0.0.0;
```

Comment:
Please use a more descriptive name for the test, like `basicRecoveryAtStatementLevel` or `statement`